package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fcp-data-access-layer-api",
-  "version": "1.3.10",
+  "version": "1.3.12",
```

Comment:
I'm pretty sure this is a feature (minor bump) and not a patch :mag:

Reply:
I mean... maybe, although all i have done is add some logs... and added a few health checks